api/blueprints/drbEdition.py

```diff
@@ -39,3 +42,7 @@ def editionFetch(editionID):
     except Exception as e: 
         logger.error(e)
         return APIUtils.formatResponseObject(500, RESPONSE_TYPE, { 'message': 'Unable to fetch edition with id {}'.format(editionID) })
+
+
+def isValidEditionID(editionID) -> bool:
+    return isinstance(editionID, int) or (isinstance(editionID, str) and editionID.isdigit())
```

Comment:
This is a good method to add to ensure the editionID is a integer even in string format.